docs/topics/development/setup_and_configuration.md

```diff
@@ -45,33 +45,33 @@ including volumes, containers, and networks. It is meant to be run frequently wh
 
 Here’s a high-level overview of what _make up_ does:
 
-```yaml
-up: setup docker_mysqld_volume_create docker_extract_deps docker_compose_up
+```{literalinclude} ../../../Makefile-os
+:lines: 165-166
```

Comment:
including literal line numbers is risky - we could easily insert/delete lines above 165 in that file.

Reply:
imo just copy the lines over and add a comment in the Makefile-os file to change this file too if changes are made there.